kpi/serializers/v2/asset.py

```diff
@@ -304,6 +324,23 @@ def get_assignable_permissions(self, asset):
             }
             for codename in asset.ASSIGNABLE_PERMISSIONS_BY_TYPE[asset.asset_type]]
 
+    def get_subscribers_count(self, asset):
+        if asset.asset_type != ASSET_TYPE_COLLECTION:
+            return 0
+
+        return UserAssetSubscription.objects.filter(asset_id=asset.pk).count()
```

Comment:
if this feature becomes popular, we might need to cache this, perhaps inside `summary`